tests/jobs/rft/data/norne/norne.py

```diff
@@ -0,0 +1,87 @@
+from itertools import product
```

Comment:
How is this test run? Or; is it a test, or something used in testing?

Could the answer be included in the file itself as a comment?

Reply:
This was the script created to generate the data required as input for the `gen_data` job. We made it early on to test both implementations.

This doesn't generate the input completely though, the zone descriptions have been added manually (e.g. `zone2_and_3`). So some modifications would be required to have the script completely generate everything.

I'm not sure if I would like this script to generate the test data at each run, or have them pre-generated (as they are now), and remove this script.

Either way, I don't think we need both

Reply:
Okay, then let's remove it.

Reply:
removed